addons/website_slides/models/slide_slide.py

```diff
@@ -939,7 +939,7 @@ def _action_set_quiz_done(self, completed=True):
         """
         if any(not slide.channel_id.is_member or not slide.website_published for slide in self):
             raise UserError(
```

Comment:
I would separate the two conditions to have distinct error message or change the message to "...if you are not among its member or if it's unpublished"